rules/falco_rules.yaml

```diff
@@ -2281,7 +2281,7 @@
   tags: [network, k8s, container, mitre_port_knocking]
 
 - list: network_tool_binaries
-  items: [nc, ncat, nmap, dig, tcpdump, tshark, ngrep]
+  items: [nc, ncat, nmap, dig, tcpdump, tshark, ngrep, telnet, ssh, mitmproxy, socat]
```

Comment:
`ssh` might be false positive prone as the list is also used in the rule `Launch Suspicious Network Tool on Host`.

Reply:
@Kaizhe Thank you.
and very sorry 🙇.  Because I re-forked this repo, I Couldn't push new commit to this PR.
So I recreated new PR. 
https://github.com/falcosecurity/falco/pull/975